tiff_2_png/tiff_2_png_converter.py

```diff
@@ -7,7 +7,6 @@
 import imagesize
 import math
 import tifffile as tiff
-import numpngw
 import image_process_helper as iph
```

Comment:
```suggestion
from tiff_2_png import image_process_helper as iph
```
does this fix your problem? I was just looking at the titan imports by our commandline functions and that's how the imports look there

Reply:
Yes! That works!!

Reply:
That should fix it. That's also how I include in my packages [here](https://github.com/brown-bnc/xnat-tools/blob/a3b5dccde58270536573cfa3e340bc0b77b0cbd6/xnat_tools/xnat2bids.py#L7)